calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -44,6 +34,9 @@ When using the Amazon VPC CNI plugin, {{prodname}} does not support enforcement
 
 :::
 
+***Prerequisites***
+
+* You [disabled network policy for the AWS VPC CNI](https://docs.aws.amazon.com/eks/latest/userguide/network-policy-disable.html). 
```

Comment:
```suggestion
* You must [disable network policy for the AWS VPC CNI](https://docs.aws.amazon.com/eks/latest/userguide/network-policy-disable.html). 
```

Reply:
Thanks, but I'm going to stick with this format. The idea of 'must' is already there, given that this is a prerequisite section.